jp2_online/urls.py

```diff
@@ -14,8 +14,17 @@
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
 from django.conf.urls import url, include
+from django.contrib.auth import views as auth_views
 from django.contrib import admin
 
+urls_password = [
```

Comment:
Maybe all of this should go inside perfiles_usuario to make it more clean?? @csssaz  @AndreeToledo  What you guys think?